src/providers/common.ts

```diff
@@ -138,7 +143,12 @@ export class Os {
   /**
    * @internal
    */
-  public static readonly _ALL_LINUX_VERSIONS = [Os.LINUX, Os.LINUX_UBUNTU, Os.LINUX_AMAZON_2, Os.LINUX_AMAZON_2023];
+  public static readonly _ALL_LINUX_VERSIONS = [Os.LINUX, Os.LINUX_UBUNTU, Os.LINUX_UBUNTU_2404, Os.LINUX_AMAZON_2, Os.LINUX_AMAZON_2023];
+
+  /**
+     * @internal
+     */
+  public static readonly _ALL_LINUX_UBUNTU_VERSIONS = [Os.LINUX_UBUNTU, Os.LINUX_UBUNTU_2404];
```

Comment:
Since we're adding Ubuntu, would be cool to have one for Amazon Linux as well since it's often tested together.